src/ape/api/providers.py

```diff
@@ -161,6 +161,19 @@ def get_code(self, address: str) -> bytes:
             bytes: The contract byte-code.
         """
 
+    @raises_not_implemented
+    def get_storage_at(self, address: str, slot: int) -> bytes:
+        """
+        Gets the raw value of a storage slot of a contract.
+
+        Args:
+            address (str): The address of the contract.
+
+        Returns:
+            bytes: The value of the storage slot.
+        """
+        raise NotImplementedError("get_storage_at is not implemented by this provider")
```

Comment:
No need for this with `raises_not_implemented` decorator!

Reply:
Also, I think Web3Provider can implement this one by default maybe, which geth also inherits